tests/unit/editors/Bugs/tests.html

```diff
@@ -1195,15 +1204,30 @@
 				equal(editor.igNumericEditor("value"), 20, "value should be 20");
 				editor.igNumericEditor("option",'value', null);
 				equal(editor.igNumericEditor("value"), 20, "value should be 20");
-				
-				
-				
-				
-
 				$("#646Editor").remove();
 			});
+			testId = 'Bug 1090';
+			test(testId, 3, function(){
+				var editor = $("#1090Editor"),
+				field = editor.igTextEditor("field"),
+				textarea = field[0],
+				text = "add some text";
+				typeInInput(text, field);
+				keyInteraction(13, field);
+				var inputSelection = editor.igTextEditor("getSelectedText"),
```

Comment:
There should really be a timeout (matching the selection one used by the editor) if you want to check the resulting selection:
```js
stop();
setTimeout(function () {
	start();
	//getSelectedText here and rest of the test
}, 100);
```
Make sure it fails if you comment out your fix.